util/reconciler/defaults.go

```diff
@@ -30,6 +30,13 @@ const (
 	DefaultMachineControllerWaitForRunningDelay = 5 * time.Second
 	// DefaultMachineControllerWaitForRunningTimeout is the default timeout if instance is not running.
 	DefaultMachineControllerWaitForRunningTimeout = 20 * time.Minute
+	// DefaultMachineControllerLinodeImage default image.
+	DefaultMachineControllerLinodeImage = "linode/ubuntu23.10"
```

Comment:
are there any concerns around us having to maintain updates on this mapping? if we are ok with this default perhaps we should use a LTS image so we have longer support?